imap_processing/cli.py

```diff
@@ -11,10 +11,11 @@
 """
 
 import argparse
-import os
 import sys
 from abc import ABC, abstractmethod
+from json import loads
```

Comment:
I find "loads" to be pretty generic and not knowing where it comes from is confusing. I'd slightly prefer this to be: `import json` and then use it as `json.loads()` where needed.